scripts/attachSessionHelper.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var fs = require('fs'),
+    http = require('http');
+
+var postData = JSON.stringify({'desiredCapabilities': {'browserName': 'chrome'}});
+var options = {
+  hostname: 'localhost',
+  port: 4444,
+  path: '/wd/hub/session',
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/x-www-form-urlencoded',
+    'Content-Length': Buffer.byteLength(postData)
+  }
+};
+var TEST = {};
+TEST.body = '';
+
+var setBody = function(chunk) {
+  TEST.body += chunk.toString();
+  var sessionId = JSON.parse(TEST.body).sessionId;
+  console.log('created new session with id: ' + sessionId);
+  fs.writeFile('spec/attachSessionTemp.js',
```

Comment:
And, if we use child process, we can intelligently wait for protractor to be done instead of using an arbitrary timeout.